File: codigos/1 - estrutura sequencial/1013 - O Maior/Main.dart
import 'dart:io';
import 'dart:math';

main(){
  List<String> entrada = stdin.readLineSync()!.split(' ');

  int A = int.parse(entrada[0]);
  int B = int.parse(entrada[1]);
  int C = int.parse(entrada[2]);

  int maior = max(max(A,B),C);

  stdout.writeln("$maior eh o maior");
}

File: codigos/1 - estrutura sequencial/1013 - O Maior/Main.cs
using System; 
using static System.Console;

// class URI {
    // static void Main(string[] args) { 
        string[] linha = ReadLine().Split(' ');

        int A = int.Parse(linha[0]);
        int B = int.Parse(linha[1]);
        int C = int.Parse(linha[2]);

        int maior = Math.Max(Math.Max(A, B), C);

        WriteLine($"{maior} eh o maior");
    // }
// }